test/WebSites/LocalizationWebSite/Models/User.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+
+namespace LocalizationWebSite.Models
+{
+    public class User
```

Comment:
This will not work today because of a bug in https://github.com/aspnet/Localization/blob/dev/src/Microsoft.Framework.Localization/ResourceManagerStringLocalizerFactory.cs#L53

I filed a bug https://github.com/aspnet/Localization/issues/90